runelite-api/src/main/java/net/runelite/api/Boss.java

```diff
@@ -0,0 +1,49 @@
+package net.runelite.api;
+
+public enum Boss
+{
+    CORPOREALBEAST("Corporeal Beast"),
```

Comment:
Generally spaces are replaced by underscores, so all these would be e.g. `CORPOREAL_BEAST`